src/SFA.DAS.ProviderApprenticeshipsService.ContractAgreements.WebJob/Configuration/ContractFeedConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.ContractAgreements.WebJob.Configuration
+{
+    public class ContractFeedConfiguration : IConfiguration
+    {
```

Comment:
Only to use ConfigurationPolicy in
SFA.DAS.ProviderApprenticeshipsService.ContractAgreements.WebJob.DependencyResolution.IoC
...